code/game/objects/items/tcg/tcg.dm

```diff
@@ -41,12 +41,38 @@ GLOBAL_LIST_EMPTY(cached_cards)
 	if(!temp)
 		return
 	name = temp.name
-	desc = temp.desc
+	desc = "<i>[temp.desc]</i>"
 	icon = icon(temp.icon)
 	icon_state = temp.icon_state
 	id = temp.id
 	series = temp.series
 
+// This totally isn't overengineered to hell, shut up
+/****
+	* Alright so some brief details here, we store all "immutable" (Think like power) card variables in a global list, indexed by id
+	* This proc gets the card's associated card datum to play with
+	*/
+/obj/item/tcgcard/proc/extract_datum()
+	var/list/cached_cards = GLOB.cached_cards[series]
+	if(!cached_cards)
+		return null
+	if(!cached_cards["ALL"][id])
```

Comment:
```suggestion
	if(!LAZYACCESSASSOC(cached_cards, "ALL", id))
```
You'll need to add parentheses to `LAZYACCESSASSOC` as well.

Reply:
The list isn't lazy, is this just cause the define makes things cleaner?

Reply:
Well, to be fair if you want it to runtime if no `"ALL"` index is found, keeping it this way is fine.

Reply:
that is what I'm after here, if the structure of that list is busted I want it to be clear, and I want to know